compiler/rustc_target/src/abi/call/mod.rs

```diff
@@ -779,26 +779,31 @@ impl RiscvInterruptKind {
 /// Metadata describing how the arguments to a native function
 /// should be passed in order to respect the native ABI.
 ///
+/// The signature represented by this type may not match the MIR function signature.
+/// Certain attributes, like `#[track_caller]` can introduce additional arguments, which are present in [`FnAbi`], but not in[`rustc_middle::ty::FnSig`].
```

Comment:
```suggestion
/// Certain attributes, like `#[track_caller]` can introduce additional arguments, which are present in [`FnAbi`], but not in [`rustc_middle::ty::FnSig`].
```

Also, I don't think `rustc_middle::ty::FnSig` is in scope?

Reply:
Just say `FnSig`, no need to intra-doc-link it.

Reply:
This may be as stupid question, but why not intra-doc it? Do intra-docs introduce some issues?

Reply:
Yes, the CI failure.

Reply:
Sorry, I somehow missed that :).